CHANGES.rst

```diff
@@ -4,7 +4,10 @@ Changelog
 1.5.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Improve the navigation endpoint by being able to return the items navigation
+  tree providing a `navigation.depth` parameter. Borrowed and adapted code from
```

Comment:
@sneridagh why do we use navigation.depth and not just "depth"? Did you try if that works with expansion as well?